frontend/plugins/payment_ui/src/modules/settings/payment/components/PaymentColumns.tsx

```
@@ -31,6 +31,7 @@ import PaymentForm from './PaymentForm';
 export const paymentColumns: ColumnDef<IPayment>[] = [
   {
     id: 'more',
+    size: 33,
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)